flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -106,6 +106,7 @@
 	/**
 	 * This option specifies the failover strategy, i.e. how the job computation recovers from task failures.
 	 */
+	@Documentation.ExcludeFromDocumentation
```

Comment:
Do we maybe want to include a reason for why this is hidden similar to deprecation notices?

Reply:
Good idea. Will change it.